docs/content/latest/yugabyte-platform/install-yugabyte-platform/install-software/openshift.md

```diff
@@ -0,0 +1,292 @@
+
+To install the Yugabyte Platform Operator, execute the following command:
+
+```shell
+oc apply -f - <<EOF
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: yugabyte-platform-operator-bundle
+  namespace: openshift-operators
+spec:
+  channel: alpha
+  name: yugabyte-platform-operator-bundle
+  source: certified-operators
+  sourceNamespace: openshift-marketplace
+EOF
+# output
+subscription.operators.coreos.com/yugabyte-platform-operator-bundle created
+```
+
+This creates a Subscription object and installs the operator in the cluster.
+
+To verify that the operator pods are in Running state, execute the following command:
+
+```shell
+oc get pods -n openshift-operators | grep -E '^NAME|yugabyte-platform'
+# output
+NAME                                                         READY  STATUS  RESTARTS  AGE
+yugabyte-platform-operator-controller-manager-7485db7486-6nzxr 2/2  Running  0      5m38s
+```
+
+For additional information, see [Adding Operators to a cluster](https://docs.openshift.com/container-platform/4.6/operators/admin/olm-adding-operators-to-cluster.html).
+
+## Finding the Availability Zone Labels
+
+You need to find the region name and availability zone codes where the cluster is running. This information is required by Yugabyte Platform (see [Creating a provider in Yugabyte Platform](https://docs.google.com/document/d/1r8V1PkZGNOcnJMsx-vz0jQvRPpVTx8AulMn7Pa1LpoM/edit?ts=602fce22#heading=h.hqqo8w43w706)). For example, if your OCP cluster is in the US East, then the cloud provider's zone labels can be us-east4-a, us-east4-b, and so on.
+
+You can use the OpenShift web console or the command line to search for the availability zone codes.
+
+### How to Use the OpenShift Web Console
+
+You start by logging in the OCP's web console as admin user, and then performing the following:
+
+- Navigate to **Compute > Machine Sets** and open each Machine Set.
+
+- On the **Machine Set Details** page, open the **Machines** tab to access the region and availability zone label, as shown in the following illustration where the region is US East and the availability zone label is us-east4-a.
+
+  ![img](https://lh3.googleusercontent.com/lIT9CxHBqoY1To2LPOYBGPpjPh7I5B4vQuZOiIK-XWy4ufsQpVEe-hTBzoADDxtTZfKwO0L7I2WlB5Fm3d5Dzreoj0z4_91WhCY2isTDxDzv5baAe0fbRK8J58vs8v1BCAygt84P)
+
+- Log out of the admin account and login with your user account.
+
+### How to Use the Command Line
+
+Alternatively, you can find the availability zone codes via the command line.
+
+You start by configuring oc with an admin account (kube:admin) and following the procedure described in [Configuring oc with the OCP cluster](https://docs.google.com/document/d/1r8V1PkZGNOcnJMsx-vz0jQvRPpVTx8AulMn7Pa1LpoM/edit?ts=602fce22#heading=h.itkxhpp0uhvz).
```

Comment:
Should point to some link from docs instead of Google docs.

Reply:
Fixed